src/components/RequestCard/index.tsx

```diff
@@ -159,7 +159,7 @@ const RequestCard: React.FC<RequestCardProps> = ({ request, onTitleData }) => {
         </div>
         {request.seasons.length > 0 && (
           <div className="card-field">
-            <span className="card-field-name">
+            <span className="hidden sm:block card-field-name">
```

Comment:
```suggestion
            <span className="card-field-name">
```

Reply:
And maybe we can move "Seasons" above the status (just switch the order), to be consistent with the ordering on the request list items?  I feel like it makes more sense that way too, since the seasons are a key attribute of the request.

Reply:
I added just the seasons text but not sure if it looks good. What do you think @sct?
![image](https://user-images.githubusercontent.com/8635678/110246926-a4381600-7f37-11eb-8f59-5ceedf13145b.png)

Reply:
@OwsleyJr Just FYI, all of your screenshots in this PR display a personal e-mail address.  You might want to black those out? 😅

I don't get to make these decisions, but I personally think the "Seasons" text looks just fine 😸  It definitely looks better than just a single numbered badge when a request is only for one season!

Reply:
Oh it's just a throwaway email that I made years ago that I use for testing so not a big deal. And I do think having just the badge looked a little odd especially if it's only one season.

Reply:
Btw, do we want to modify the "Seasons" strings on the request list & cards to reflect the correct plurality?

```
seasons: '{seasonCount, plural, one {Season} other {Seasons}}',
```

```
{intl.formatMessage(messages.seasons, { seasonCount: request.seasons.length })}
```